Ayaalassaf/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+
+    </head>
+    <body>
+        <form "/action_page.php" method="post">
+            <label for="email">Enter your email:</label><br>
+            <input type="email" id="email" name="email" required><br><br>
+            <label for="fname">Username:</label><br>
+            <input type="text" id="fname" name="fname"><br><br>
```

Comment:
after each input label pair put new line "Empty Line"

        <label for="email" >enter your email</label><br> 
        <input type="email" id="email" name="email" value="john@example.com" required ><br>

        <label for="fname">Username</label><br>
        <input type="text" id="fname" name="fname" value="osama"><br>

because it will be more readable.